core/imds/pom.xml

```diff
@@ -105,16 +105,6 @@
             <artifactId>test-utils</artifactId>
             <version>${awsjavasdk.version}</version>
             <scope>test</scope>
-            <exclusions>
```

Comment:
removed this because `import software.amazon.awssdk.testutils.EnvironmentVariableHelper; ` had dependency on Junit4  `import org.junit.rules.ExternalResource;
`